### PR TITLE
Update tts_de.py

### PR DESCRIPTION
**Pull request by rcbebo82**
Translated crow to "Butterfly", changed filenames accordingly to the english ones so its possible to change from english to german without the need of editing the Special Functions section per model.
Removed "Minimale Trimmung", in my opinion this makes no sense a trim only can be at maximum or at center.

### CODE REVIEW

File: radio/util/tts_de.py [hunk 1]
```diff
@@ -50,8 +50,8 @@
                 (u"Fehler", "error", NO_ALTERNATE),
                 (u"Trim zentriert", "midtrim", 495),
                 (u"Poti zentriert", "midpot", 496),
-                (u"Maximale Trimmung erreicht", "maxtrim", NO_ALTERNATE),
-                (u"Minimale Trimmung erreicht", "mintrim", NO_ALTERNATE),
```
Comment (kilrah) [inline radio/util/tts_de.py]: There need to be 2 files, mintrim and maxtrim. You can have them both say the same thing.

Comment (3djc) [inline radio/util/tts_de.py]: Also you cannot change the name of a bunch of sounds like that, many people have referenced them in special fuincitons that will suddenyl stop working

Comment (kilrah) [inline radio/util/tts_de.py]: It's a system sound, no special functions.

Comment (rcbebo82) [inline radio/util/tts_de.py]: So it would be no problem if for those two files the same is spoken. Should i correct it?

**Comment by schwabe** [comment]
I think the information which limit of trimming you reached is still useful. If you don't like Mimimum and Maximum, we should go with:

"Untere Trimmungsgrenze erreicht" and
"Obere Trimmungsgrenze erreicht" 

oder something similar.

**Comment by rcbebo82** [comment]
Got it now i understand. Mintrim is on the Zero side of each Input so it makes Sense. But what is with the filemames?

**Comment by schwabe** [comment]
Just keep the filenames mintrim and maxtrim.

**Comment by 3djc** [comment]
Thanks a lot

**Pull request merged**